### DISCUSSION TITLE
Is it free to change address?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9289] [General Q&A] [answered]
I plan to change my IP. Through the "is it free to change address XXX. XXX. XXX. XXX" command, do I need to pay fil?

**Comment by stuberman** [comment]
Any time you put a message on chain there is a gas fee, so yes.

**Reply by rjan90** [reply]
Beat me to it by a couple of seconds 😄

**Comment by rjan90** [comment, marked as answer]
Hey @ChaiWenyu1983! Changing the PeerID will send a message to the chain which will cost a small amount of FIL. For the past 24 hours, there has been 3 messages like that sent to the chain, with an average cost of 0.0003 FIL. You check the costs for different messages here: https://filfox.info/en/stats/gas
